package.json

```diff
@@ -29,7 +29,6 @@
   "scripts": {
     "build": "gatsby build",
     "develop": "gatsby develop",
-    "dev": "yarn develop",
```

Comment:
Dup of `yarn develop`

Reply:
`yarn dev` is what we use in other projects so maybe we should get rid of `develop`? Of course that's there because that's what Gatsby puts down by default. Happy for you to make the call here..

Reply:
I'm not too picky on this -- just think it makes sense to only have 1. Since `dev` is used in other projects and it is shorter, let's just go w/ that. I'll update.